AtalaPrismSDK/PrismAgent/Sources/PrismAgent+Backup.swift

```diff
@@ -0,0 +1,356 @@
+import Domain
```

Comment:
Should this PR contains logic for the backup?!

Reply:
Yes it was a mistake already removed.